web/src/e2e/index.e2e.test.tsx

```diff
@@ -364,6 +366,101 @@ describe('e2e test suite', function(this: any): void {
+                    },
+                    {
+                        filePath:
+                            '/github.com/sourcegraph/go-diff@3f415a150aec0685cb81b73cc201e762e075006d/-/blob/diff/diff.go',
+                        symbolNames: [
+                            'diff',
+                            'Stat',
+                            'Stat',
+                            'hunkPrefix',
+                            'hunkHeader',
+                            'diffTimeParseLayout',
+                            'diffTimeFormatLayout',
+                            'add',
+                        ],
+                        symbolTypes: [
+                            'package',
+                            'function',
+                            'function',
+                            'variable',
+                            'constant',
+                            'constant',
+                            'constant',
+                            'function',
+                        ],
+                    },
+                    {
+                        filePath:
+                            '/github.com/sourcegraph/appdash@ebfcffb1b5c00031ce797183546746715a3cfe87/-/blob/python/appdash/sockcollector.py',
+                        symbolNames: [
+                            'RemoteCollector',
+                            'sock',
+                            '_debug',
+                            '__init__',
+                            '_log',
+                            'connect',
+                            'collect',
+                            'close',
+                        ],
+                        symbolTypes: ['class', 'variable', 'variable', 'field', 'field', 'field', 'field', 'field'],
+                    },
+                    {
+                        filePath:
+                            '/github.com/sourcegraph/sourcegraph-typescript@a7b7a61e31af76dad3543adec359fa68737a58a1/-/blob/server/src/cancellation.ts',
+                        symbolNames: [
+                            'createAbortError',
+                            'Object',
+                            'isAbortError',
+                            'throwIfCancelled',
+                            'tryCancel',
+                            'toAxiosCancelToken',
+                            'source',
+                        ],
+                        symbolTypes: [
+                            'constant',
+                            'constant',
+                            'constant',
+                            'function',
+                            'function',
+                            'function',
+                            'constant',
+                        ],
+                    },
+                ]
+
+                for (const test of tests) {
+                    await driver.page.goto(baseURL + test.filePath)
+
+                    await (await driver.page.waitForSelector('.tab-bar__tab[data-e2e-tab="symbols"]')).click()
```

Comment:
Don't rely on styling CSS classes in e2e tests as they can change and shouldn't affect e2e tests. See https://sourcegraph.com/github.com/sourcegraph/sourcegraph/-/blob/doc/dev/testing.md#adding-a-new-e2e-test

Here, the attribute selector is sufficient
```suggestion
                    await (await driver.page.waitForSelector('[data-e2e-tab="symbols"]')).click()
```